### DISCUSSION TITLE
PvP Group Queueing

### DISCUSSION THREAD

**Discussion by Yovhan** [chromiecraft/chromiecraft#5665] [Suggestions]
(Disclaimer: I'm only speaking for myself and my friends)

Gotta say this about PVP:

The time when you had PvP group queue available (up to 3 people) was a blast, and made pvp an actually fun activity for me and my friends to participate in. 
Atm, with the system just randomly allocating players on the one or other team, it is so hard to actually play together, we usually end up on different teams. It is really discouraging to participate, when you can't strategize with your friends and play together, and everything is just random.
I've always enjoyed pvping with friends, and it is sad that this is taken away from the game.

I'm looking forward to arena, but dreading the honor farm for off-pieces with the current system.
Please let us sign up for Bgs with a couple of friends for each BG. Otherwise i fear, that pvp for me PVP will just not be fun to do.

If it is a balancing issue:
WSG could be 2 people, AB 3, AV ?      - or something like that, if you think 3 people for WSG is to much.

Btw it's not balanced now anyways, it's just random. random is not balance. Random is, however, so unfun and frustrating.  (meeting a team with 3 healers, and your team has no healer as 1 example)

Lastly, i just want to point out that almost every game that does PvP allows for some sort of grouping up. It is the whole point of pvp in my opinion, do you have an advantage by grouping up with friends, ofcourse you do because now you can coordinate and actually play the game as it was meant to - synergize one class with another.
This is an integral part of the MMO socializing experience, it encourages people talking together, playing together, and making connections which is a good thing. This was, back in the day, an integral and important feature of the MMO-RPG genre which made these games magical (in my humble opinion).

Hopefully you will reconsider.

in any case, please let me know what the issue is.

Thank you for your work.

**Comment by Never6** [comment]
Bgs have too small of pop for group queue 
And group queue tends to break everytime its implemented

**Comment by trinitarius** [comment]
I disagree. When I queue for BGs I ask people to queue "together" with me and if we are lucky we end up on the same team, being able to discuss tactics and try to win. When not, we have fun trying to take each other out, have duels, try to win over the other etc. It creates a lot of fun moments and experiences as well.

> The time when you had PvP group queue available (up to 3 people) was a blast

Disagree. From what I remember it was more abused by very experienced players, queueing up together with the goal of being exclusively on top of the leaderboards. Due to not so many people participating in PvP teams were usually highly unbalanced.

> Btw it's not balanced now anyways

That is to some extent true, hence there are other discussions about balancing teams better by new players automatically joining the team with less players; it is more balanced compared to the times when group queue was enabled though.

> This is an integral part of the MMO socializing experience

That includes playing against each other as well. What is the problem with fighting against your friends? In the end it is a social game to have fun in and I have fun playing with and against any player, including my friends. It creates a healthy form of rivalry within, compared to sweaty tryharding that can bring out the worst of some people.

I understand that this is your opinion and you value PvP maybe differently than I do but it is more fun to me now playing PvP with friends than when group queue was enabled.

**Reply by Yovhan** [reply]
yeah, it's just differnce in experience and how we enjoy PVP, I'd rather lose with my friends, than win against my friends :). Anyways, just wanted to get my perspective out there, i'm not at all certain how the majority feel. 

For me though, as it is now it is discouraging to participate in PVP.

**Comment by Eledor-collab** [comment]
and what about if we use the current PvP ranking system to the Group queueing to make it more friendly to casual pvpiers. For instance, you could prohibit group queueing of two ppl that have rank 6 (20k HKs) or above.

This might prevent infamous BGs with "OP" people queueing together all the time. Take the case of a very pvp-active and strong warrior with a unkillable holy priest... that would turn WSG at night into a situation similar to BGs in the 10-19 bracket. "The team that gets the hunter twink is the team that wins".
